site/learn/expert-advice/authentication/webauthn-explained.md

```diff
@@ -0,0 +1,478 @@
+---
+layout: advice
+title: WebAuthn explained
+description: An in-depth look at how WebAuthn can help you add biometric authentication to your website.
+author: Dan Moore
+image: advice/vendor-lockin/expert-advice-avoiding-authentication-system-lock-in-header-image.png
+category: Authentication
+date: 2022-09-28
+dateModified: 2022-09-28
+---
+
+Webauthn is a new way to authenticate your users using biometric, secure authentication methods. WebAuthn requires something called an authenticator. What is an authenticator? An authenticator is a hardware or software device that is used to authenticate a user, via biometric or other methods, and that follows certain standards.
+
+That's a bit tautological, isn't it? Well, yes, but read on for more details. Also, clowns.
+
+## WebAuthn Overview
+
+WebAuthn has flows of data and interaction, termed "ceremonies". These extend a network protocol to include things in the real world, such as physical hardware or human beings. There are three main ones:
+
+1. Registration, where the user associates an authenticator with their account
+1. Authentication, where the user logs in, using WebAuthn
+1. Attestation, where the authenticator is tied back to a known source, such as an operating system vendor
```

Comment:
Attestation is not its own [ceremony](https://www.w3.org/TR/webauthn-2/#ceremony) actually. It's an optional part of the Registration ceremony.

Reply:
Doh, thanks!